pom.xml

```diff
@@ -91,6 +91,36 @@
                     <encoding>${project.build.sourceEncoding}</encoding>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <artifactId>maven-release-plugin</artifactId>
+                <configuration>
+                    <useReleaseProfile>false</useReleaseProfile>
+                    <autoVersionSubmodules>true</autoVersionSubmodules>
+                    <tagNameFormat>@{project.version}</tagNameFormat>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>io.zipkin.centralsync-maven-plugin</groupId>
+                <artifactId>centralsync-maven-plugin</artifactId>
+                <configuration>
+                    <subject>opentracing</subject>
+                    <repo>maven</repo>
+                    <packageName>io.opentracing.contrib.hprose</packageName>
```

Comment:
IIRC this is a package name in Bintray. It might be good to be consistent and use something like
`opentracing-hprose` or `java-hprose`

Maybe `java-hprose` is the best as it express a direct correlation with repo, although `io.opentracing.contrib.hprose` is fine, but I'm mot sure if `okhttp` encodes path params.

https://github.com/openzipkin/centralsync-maven-plugin/blob/master/src/main/java/zipkin/maven/CentralSyncMojo.java#L81
https://square.github.io/okhttp/2.x/okhttp/com/squareup/okhttp/HttpUrl.html

I would recommend to do not use `.` here.